package.json

```diff
@@ -92,9 +93,7 @@
     "build-dist": "bash ./scripts/build-dist.sh",
     "build-win-installer": "scripts\\build-windows-installer.bat",
     "check-lockfile": "./scripts/check-lockfile.sh",
-    "lint": "yarn run lint-prettier && eslint . && flow check",
-    "lint-prettier": "node scripts/prettier.js lint",
-    "prettier": "node scripts/prettier.js write",
```

Comment:
You are right I forgot to delete the script, i will do asap. Yes, it is that the intetion, so everything will be like before I added prettiee :)